src/App.jsx

```diff
@@ -16,31 +13,33 @@ import { VanDetails as HostVandetails } from "./pages/Host/VanDetails";
 import { SingleVanDetails as Details } from "./components/vans/SingleVanDetails";
 import { Pricing } from "./components/vans/Pricing";
 import Photos from "./components/vans/Photos";
+import Layout from "./components/Layout";
+import Page404 from "./pages/Page404";
 
 function App() {
     return (
         <BrowserRouter>
-            {/* #TODO: Move navigation inside the layout route */}
-            <Navigation />
             <Routes>
-                <Route path="/" element={<HomePage />} />
-                <Route path="/about" element={<About />} />
-                <Route path="/vans" element={<VansList />} />
-                <Route path="/vans/:id" element={<VanDetails />} />
-                <Route path="/host" element={<HostLayout />}>
-                    <Route index element={<Dashboard />} />
-                    <Route path="income" element={<Income />} />
-                    <Route path="reviews" element={<Reviews />} />
-                    <Route end path="vans" element={<HostVans />} />
-                    <Route path="vans/:id" element={<HostVandetails />}>
-                        <Route index element={<Details />} />
-                        <Route path="details" element={<Details />} />
-                        <Route path="pricing" element={<Pricing />} />
-                        <Route path="photos" element={<Photos />} />
+                <Route path="/" element={<Layout />}>
+                    <Route path="/" element={<HomePage />} />
```

Comment:
Turning this route into an index route would be better than using an absolute path.